jraft-rheakv/rheakv-core/src/main/java/com/alipay/sofa/jraft/rhea/errors/Errors.java

```diff
@@ -36,7 +36,10 @@
  * Do not add exceptions that occur only on the client or only on the server here.
  */
 public enum Errors {
-    UNKNOWN_SERVER_ERROR(-1, "The server experienced an unexpected error when processing the request",
+    RPC_CONNECTION_ERROR(-2, "RPC connection disconnected.",
```

Comment:
It is better to use positive error codes, leaving negative only for UNKNOWN_SERVER_ERROR.